tests/integration_tests/build/test_coverage.py

```diff
@@ -31,7 +31,7 @@ def is_on_skylake():
     }
 else:
     COVERAGE_DICT = {
-        "Intel": 78.90 if is_on_skylake() else 79.52,
```

Comment:
How does this change the code coverage?

Reply:
I guess that CI tests for a past PR didn't get Intel skylake.

Reply:
This is correct. The previous time these [tests were run to adjust coverage they ran on CL](https://buildkite.com/firecracker/firecracker-pr/builds/2452#0187b8ee-97bb-4091-b124-bdd265c756c6),
```
Model name:           Intel(R) Xeon(R) Platinum 8259CL CPU @ 2.50GHz
```